src/registry/select/select-validation.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import userEvent from '@testing-library/user-event';
+
+import ComponentEditForm from '@/components/ComponentEditForm';
+import {contextRender, screen} from '@/test-utils';
+
+describe('Manual values: must have at least one non-empty value', () => {
+  const component = {
+    id: 'wqimsadk',
+    type: 'select',
+    key: 'select',
+    label: 'A select field',
+    openForms: {
+      dataSrc: 'manual',
+      translations: {},
+    },
+    values: [{value: '', label: ''}],
+    defaultValue: '',
+  };
+  const builderInfo = {
+    title: 'Select',
+    icon: 'th-list',
+    group: 'basic',
+    weight: 70,
+    schema: {},
+  };
```

Comment:
I can't wait for this to be centralized :)

You can add some type annotations though: `const component = {...} satisfies SelectComponentSchema;`

Reply:
Hum, `Object literal may only specify known properties, and 'values' does not exist in type 'SelectComponentSchema'`. Apparently `values` should be nested under `data` according to the type definition. But I've just copied the structure from the SB test, so it might be invalid there as well?

Reply:
It likely is - the component edit form itself applies default values which may shadow this issue then. Please do check it.